### PR TITLE
Step missing

### PR DESCRIPTION
**Pull request by joinimran**
There was a step missing that has been added in the doc.

Problem: https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/748

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/integrate-yammer-with-other-apps/yammer-and-newsfeed.md [hunk 1]
```diff
@@ -39,8 +39,10 @@ You can turn Yammer off or on for conversations in SharePoint by using the Share
 3. In the left pane, choose **Admin centers** \> **SharePoint**.
     
 4. Choose **Settings** on the left nav. 
+
+5. Click on **classic settings page**.
```
Comment (JohanFreelancer9) [inline yammer/integrate-yammer-with-other-apps/yammer-and-newsfeed.md]:
```suggestion
5. Choose **Classic settings page**.
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@TeresaFG-writer - PR has been copy edited and is ready for final review. This is based on issue #748

Thank you.

cc: @kenwith

**Comment by TeresaFG-writer** [comment]
@joinimran Looks good.

**Pull request merged**